basecli/k8s.go

```diff
@@ -39,26 +47,26 @@ func runGetK8sCmd(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
-func Generate(values []string) (result *bytes.Buffer, err error) {
-	v := chartutil.Values{}
+// Generate the k8s experiment manifest
+func Generate() (result *bytes.Buffer, err error) {
+	p := getter.All(cli.New())
+	v, err := GenOptions.MergeValues(p)
+	if err != nil {
+		return nil, err
+	}
 
 	// add id=id if --id option used
 	// note that if both --id=foo and --set id=bar are used,
 	// the --id option will take precedence
-	if len(id) > 0 {
-		values = append(values, "id="+id)
+	if len(Id) > 0 {
+		v["id"] = Id
 	}
 
 	// add app=app if --app option is used
-	// not that if both --app=foo and --set id=bar are used,
-	// the --app optiom will take precedence
-	if len(app) > 0 {
-		values = append(values, "app="+app)
-	}
-
-	err = ParseValues(values, v)
-	if err != nil {
-		return nil, err
+	// note that if both --app=foo and --set id=bar are used,
```

Comment:
documentation should be  `.... --set app=bar are used.`

Reply:
Fixed.

Reply:
Nothing is ignored in `iter8 gen k8s`. Will push a change to reflect this shortly.

Reply:
Fixed now.